docs/testing/debug-add-ins-using-devtools-edge-legacy.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Debug add-ins using developer tools for Microsoft Edge Legacy
+description: 'Debug add-ins using the developer tools in Microsoft Edge Legacy'
+ms.date: 10/20/2021
+ms.localizationpriority: medium
+---
+
+# Debug add-ins using developer tools in Microsoft Edge Legacy
+
+This article shows how to debug the client-side code (JavaScript or TypeScript) of your add-in when the following conditions are met:
+
+- You cannot, or don't wish to, debug using tools built into your IDE; or you are encountering a problem that only occurs when the add-in is run outside the IDE.
+- Your computer is using a combination of Windows and Office versions that use the original Edge webview control, EdgeHTML.
+
+To determine which browser is being used on your computer, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). 
+
+> [!TIP]
+> [!INCLUDE[Identify the webview through the add-in UI](../includes/identify-webview-in-ui.md)]
+
+> [!NOTE]
+> To install a version of Office that uses the Edge legacy webview, see [Install a version of Office that uses Microsoft Edge Legacy](#install-a-version-of-office-that-uses-microsoft-edge-legacy).
+
+## Debug a task pane add-in using Microsoft Edge DevTools Preview
+
+1. Install the [Microsoft Edge DevTools Preview](https://www.microsoft.com/p/microsoft-edge-devtools-preview/9mzbfrmz0mnj?activetab=pivot%3Aoverviewtab). (The word "Preview" is in the name for historical reasons. There is no more recent version.)
+
+   > [!NOTE]
+   > If your add-in has an [Add-in Command](../design/add-in-commands.md) that executes a function, the function runs in a hidden browser process that the Microsoft Edge DevTools cannot detect or attach to, so the technique described in this article cannot be used to debug code in the function.
+
+1. [Sideload](create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) and run the add-in.
+1. Run the Microsoft Edge DevTools.
+1. In the tools, open the **Local** tab. Your add-in will be listed by its name. (Only processes that are running in EdgeHTML appear on the tab. The tool cannot attach to processes that are running in other browsers or webviews, including Microsoft Edge (WebView2) and Internet Explorer (Trident).)
+
+   ![Screenshot of Edge DevTools showing a process named legacy-edge-debugging.](../images/edge-devtools-with-add-in-process.png)
+
+5. Select the add-in name to open it in the tools.
+6. Open the **Debugger** tab.
+7. Open the file that you want to debug with these steps:
+
+   <ol type="a">
```

Comment:
Why is this in HTML? Shouldn't it be formatted as a numbered list procedure?

Reply:
https://docs.microsoft.com/en-us/contribute/markdown-reference#numbered-list
This link describes how to create nested lists with the outer list numerals and the sub-list alpha.